### DISCUSSION TITLE
怎么实现用ConfigProvider token控制Table组件中的pagination的布局

### DISCUSSION THREAD

**Discussion by eveforest** [ant-design/ant-design#42661] [Q&A] [answered]
我想用token自定义pagination的布局大小，但是在Table组件中pagination是一个属性，只能配置已定义的值，有没有人能告诉我怎么做？

**Comment by BoyYangzai** [comment]
@eveforest 布局具体指的是？宽高？

**Reply by eveforest** [reply]
对，宽高，还有可以的话其他design token也想尝试

**Reply by BoyYangzai** [reply, marked as answer]
> 对，宽高，还有可以的话其他design token也想尝试

这边是一些全局的样式 可以看看有没有需要的
https://ant.design/components/pagination-cn#design-token

修改布局的话其实通过类名很简单就可以完成修改
写了个dmeo：
https://codesandbox.io/s/ji-ben-yong-fa-antd-5-5-1-forked-m0lrtc?file=/index.css

更多的组件专属Design Token我们正在努力进行迁移从v4 less 迁移过来，
当前进度：[#41884](https://github.com/ant-design/ant-design/issues/41884)
也欢迎你参与到迁移的贡献中来🥰

**Reply by eveforest** [reply]
我在尝试把布局等设计控制放到js中来而不使用less，这样可以通过全局store状态动态控制布局变化。
迁移的工作我先学习了解下，不一定有足够的时间参与进去，感谢邀请。
